Makefile

```diff
@@ -433,7 +434,14 @@ TARGETS = \
 	installers-all \
 	electron-develop
 
-package-electron: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH)
```

Comment:
Oh yeah, I hadn't spotted that `$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-linux-$(TARGET_ARCH)` _is_ still being used :)